src/main/java/com/minecolonies/coremod/colony/buildings/BuildingBlacksmith.java

```diff
@@ -54,19 +54,19 @@ public int getMaxBuildingLevel()
         return MAX_BUILDING_LEVEL;
     }
 
+    //TODO Implement Later
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)